agroal/src/main/resources/org/wildfly/extension/agroal/LocalDescriptions.properties

```diff
@@ -0,0 +1,106 @@
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this software; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+# 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+#
+#
+# PLEASE KEEP DESCRIPTION IN SYNC WITH THE XML SCHEMA
+#
+agroal=Agroal subsystem
+agroal.add=Add Operation for the Agroal subsystem
+agroal.remove=Remove Operation for the Agroal subsystem
+#
+#
+# DataSource and XADataSource operations
+agroal.flush-all=Flush all connections from the pool immediately
+agroal.flush-graceful=Flush all connections from the pool, allowing connections to be closed normally
+agroal.flush-invalid=Flush invalid connections from the pool
+agroal.flush-idle=Flush connections not being use
+agroal.statistics-reset=Reset statistics
+#
+#
+# DataSource attributes
+agroal.datasource=A datasource
+agroal.datasource.add=Add operation for a datasource resource in the model
+agroal.datasource.remove=Add operation for a datasource resource in the model
+#
+agroal.datasource.name=Name for the datasource (used for management)
+agroal.datasource.jndi-name=Specifies the JNDI name for the datasource
+agroal.datasource.jta=Enable JTA integration
+agroal.datasource.connectable=Enable CMR (Commit Markable Resource) functionality on this datasource
+agroal.datasource.statistics=Runtime statistics provided by the datasource
+agroal.datasource.statistics-enabled=Enable statistics for this datasource
+#
+agroal.datasource.connection-factory=Configuration for the connection factory
+agroal.datasource.connection-factory.driver=Unique reference to the JDBC driver
+agroal.datasource.connection-factory.url=JDBC driver connection URL (e.g. "jdbc:h2:tcp://localhost:1234")
+agroal.datasource.connection-factory.transaction-isolation=Set the java.sql.Connection transaction isolation level to used
+agroal.datasource.connection-factory.new-connection-sql=SQL statement to be executed on a connection after creation
+agroal.datasource.connection-factory.security-username=Username used when creating a new connection
+agroal.datasource.connection-factory.security-password=Password used when creating a new connection
+agroal.datasource.connection-factory.connection-properties=Properties to be passed to the JDBC driver when creating a connection
+#
+agroal.datasource.connection-pool=Configuration for the connection pool
+agroal.datasource.connection-pool.max-size=Maximum number of connections in the pool
+agroal.datasource.connection-pool.min-size=Minimum number of connections the pool should hold
+agroal.datasource.connection-pool.initial-size=Initial number of connections the pool should hold
+agroal.datasource.connection-pool.blocking-timeout-millis=Maximum time in milliseconds to block while waiting for a connection before throwing an exception
```

Comment:
you don't need to specify the measurement unit as it is included in the attribute metadata